PostProcessing/combine3dMany.py

```diff
@@ -0,0 +1,74 @@
+import glob 
```

Comment:
Both `combine3dFields.py` and `combine3dMany.py` are hard coded with some of my old simulations, so they won't work out of the box. We can keep them for reference, but can also remove them if we want to keep the codes cleaner.